CRUD_Operations.c

```diff
@@ -0,0 +1,173 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define FILE_NAME "users.txt"
+
+typedef struct {
+    int id;
+    char name[50];
+    int age;
+} User;
+
+//for adding new users
+void create_user(){
+     User user;
+    FILE *file = fopen(FILE_NAME, "a");
+    if (!file) {
+        printf("Error opening file.\n");
+        return;
+    }
+
+    printf("Enter User ID: ");
+    scanf("%d", &user.id);
+    printf("Enter Name: ");
+    scanf(" %[^\n]", user.name);  // we can use this for taking string with spaces
+    printf("Enter Age: ");
+    scanf("%d", &user.age);
+    // adding data to our file
+    fprintf(file, "%d,%s,%d\n", user.id, user.name, user.age);
+    fclose(file);
+
+    printf("Successfully Added\n");
+};
```

Comment:
Why are you using ";" after every method. Explain its use?

Reply:
It is not necessary to add this (;) as I code in a different language. I will change it and will make sure not to repeat this